parse/StartingBuildingsParser.cpp

```diff
@@ -0,0 +1,50 @@
+#include "Label.h"
+#include "Parse.h"
+#include "ParseImpl.h"
+#include "../util/Directories.h"
+
+#include <boost/spirit/include/phoenix.hpp>
+
+#define DEBUG_PARSERS 0
+
+#if DEBUG_PARSERS
+namespace std {
+    inline ostream& operator<<(ostream& os, const std::vector<ItemSpec>&) { return os; }
+}
+#endif
+
+namespace {
+    struct rules {
+        rules() {
+            qi::_1_type _1;
+            qi::_2_type _2;
+            qi::_3_type _3;
+            qi::_4_type _4;
+            qi::_r1_type _r1;
+            using phoenix::push_back;
+
+            start
+                =   +parse::detail::item_spec_parser() [ push_back(_r1, _1) ]
+                ;
+
+            start.name("start");
+
+            qi::on_error<qi::fail>(start, parse::report_error(_1, _2, _3, _4));
+        }
+
+        typedef boost::spirit::qi::rule<
+            parse::token_iterator,
+            void (std::vector<ItemSpec>&),
+            parse::skipper_type
+        > start_rule;
+
+        start_rule start;
+    };
+}
+
+namespace parse {
+    bool starting_buildings(std::vector<ItemSpec>& starting_buildings_) {
```

Comment:
does this need to be in a new file? could you just add this function to ItemsParser.cpp instead, and use the existing rule definition in that file? (assuming you want to keep it parting items and not just a list of strings...)